frontend/cypress/support/helpers.ts

```diff
@@ -6,6 +6,20 @@ export const getRadioButtonByDataTestId = (id: string) => {
     return cy.get(`[data-testid=${id}]`).children().eq(0)
 }
 
+export const getChecklistElement = (element: string) => {
+    return cy.get('li').contains(element)
+}
```

Comment:
Will this work if we have several checklists?

Reply:
Yes, and that's because we need to .click() the relevant dropdown first. I'll add a comment stating this